opencti-platform/opencti-graphql/config/schema/opencti.graphql

```diff
@@ -1735,7 +1754,9 @@ type Connector implements BasicObject & InternalObject {
   connector_schema_ui: String
   connector_state_reset: Boolean
   connector_user_id: ID
+  connector_user: User
```

Comment:
This should be protected directly here (instead of in domain) : 
`@auth(for: [SETTINGS_SETACCESSES])`